contents/blog/pricing-lessons.md

```diff
@@ -0,0 +1,96 @@
+
+**Be transparent about pricing** so users don't have to 'talk to sales'. Is the best way to close a customer to signal to them that you're going to charge them as much as you can get away with? Many people think so. Disclaimer: we started non-transparent so we could speak to some users to figure out the right model first (and we still do this - see our non-transparent enterprise pricing)... but we make it transparent as soon as we've reasonable confidence.
+
+**Be self serve** so users can start paying you to solve a problem they have _right now_. Defaulting to _not_ using a sales team, and providing transarent pricing, means you can get people in the moment.
+
+## But don't lose the ability to do big contracts
+
+* We differentiated on features
+* Our usage grows with our customers' products. If we charged a flat fee, we couldn't do this so easily.
+
+Just because it's easy to get started, doesn't mean your product has to be cheap.
+
+Some companies with usage pricing get criticized for the escalating costs at scale. If it takes just one extra engineer to self build in a large company, you theoretically shold be able to charge >$100K annually. If you've competition, it depends what they charge. 
+
+## You don't have to do all these things immediately
+
+Early on, optimize for learning. Later on, optimize for the best solution for your users.
+
+**We didn't start with a $0 minimum**. Last summer, we worked on finding just five reference customers for our paid product (we now have X customers).
+
+For these, we didn't offer $0 minimum - we charged $2k/month minimum. We simply couldn't support more than this, and we were already getting enough inbound interest.
+
+**We didn't start with self-serve sales**, as we wanted to talk to users at first. This meant we grew more slowly to optimize for learning at the start. Once we had a reasonable formula, we could accelerate things by chopping out any manual sales process.
+
+## Pricing to value is short sighted
+
+Conventional wisdom is to "price to value", which is broadly interpreted as you should charge a given customer the most you can.
+
+This gives you no room - if the service is at all below initial expectations, the customer may churn. And given you likely don't know the precise dollar value for a given customer, you'll end up losing customers for pricing reasons, or at least being a pain to negotiate with. As a VC-backed company, I'd prefer happy customers, momentum and more growth.
+
+There are exceptions to this rule - relationship-driven enterprise sales, for example. But, long term, anyone pricing like this is asking to get disrupted.
+
+## Make pricing work for your biggest customers
+
+We implemented logarithmically-declining pricing with usage. This means something like - every time your volume goes up 10x, the incremental price decrease 5x.
```

Comment:
This section feels short... is it part of the section before? i.e. we don't price to value we do this instead.

Reply:
I've adding a little more context so it doesn't feel too abrupt